### PR TITLE
Add ephemeral storage and revise

### PR DESCRIPTION
**Pull request by ritawatson**
Added ephemeral storage for virtual clusters and revamped doc. 

Jira: https://spectrocloud.atlassian.net/browse/PPD-620

[Preview](https://deploy-preview-1029--docs-spectrocloud.netlify.app/devx/resource-quota)

### CODE REVIEW

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/1029#" title="Author self-approved">ritawatson</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: content/docs/04.5-devx/04-resource-quota.md
Comment (karl-cardenas-coding) [inline content/docs/04.5-devx/04-resource-quota.md:32]:
```suggestion
1. Log in to [Palette](https://console.spectrocloud.com/). 
```

**Review by karl-cardenas-coding — APPROVED** [review]
Minor suggestion, otherwise 👍🏻

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/1029#pullrequestreview-1213712951" title="Approved">karl-cardenas-coding</a>*, *<a href="https://github.com/spectrocloud/librarium/pull/1029#" title="Author self-approved">ritawatson</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: content/docs/04.5-devx/04-resource-quota.md
Comment (ritawatson) [reply content/docs/04.5-devx/04-resource-quota.md]: fixed typo

File: content/docs/04.5-devx/04-resource-quota.md
Comment (ritawatson) [reply content/docs/04.5-devx/04-resource-quota.md:21]: fixed

File: content/docs/04.5-devx/04-resource-quota.md
Comment (ritawatson) [reply content/docs/04.5-devx/04-resource-quota.md:19]: fixed typo

**Review by tiwarisumit3 — APPROVED** [review]

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/1029#pullrequestreview-1213712951" title="Approved">karl-cardenas-coding</a>*, *<a href="https://github.com/spectrocloud/librarium/pull/1029#" title="Author self-approved">ritawatson</a>*, *<a href="https://github.com/spectrocloud/librarium/pull/1029#pullrequestreview-1214096478" title="Approved">tiwarisumit3</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**
